java/src/main/java/com/google/developers/wallet/rest/DemoEventTicket.java

```diff
@@ -287,15 +283,14 @@ public String AddClassMessage(String issuerId, String classSuffix, String header
    * @param classSuffix Developer-defined unique ID for this pass class.
    * @param objectSuffix Developer-defined unique ID for this pass object.
    * @return The pass object ID: "{issuerId}.{objectSuffix}"
-   * @throws IOException
```

Comment:
Should this be removed? The method still throws. Seeing this same thing in a few different methods